.github/workflows/gradle.yml

```diff
@@ -154,12 +157,12 @@ jobs:
           attempt_limit: 3
           attempt_delay: 5000 # in ms
 
-  cdk-build:
+  python-cdk-build:
```

Comment:
The primary reason for the change was to resolve name collision - since "cdk" is now ambiguous. Also, below, the paths needed to be updated so that the Python trigger doesn't kick off when only java-side files are modified.

But you are correct - as of now, there isn't a java CDK flow created yet.